gensim/test/test_word2vec.py

```diff
@@ -1162,15 +1164,21 @@ def test_path_line_sentences_one_file(self):
 
 # endclass TestWord2VecSentenceIterators
 
-# TODO: get correct path to Python binary
-# class TestWord2VecScripts(unittest.TestCase):
-#     def test_word2vec_stand_alone_script(self):
-#         """Does Word2Vec script launch standalone?"""
-#         cmd = 'python -m gensim.scripts.word2vec_standalone -train ' + datapath('testcorpus.txt') + \
-#               ' -output vec.txt -size 200 -sample 1e-4 -binary 0 -iter 3 -min_count 1'
-#         output = check_output(cmd, stderr=PIPE)
-#         self.assertEqual(output, '0')
-# #endclass TestWord2VecScripts
+
+class TestWord2VecScripts(unittest.TestCase):
+    def test_word2vec_stand_alone_script(self):
+        """Does Word2Vec script launch standalone?"""
+        cmd = [
+            sys.executable, '-m', 'gensim.scripts.word2vec_standalone',
+            '-train', datapath('testcorpus.txt'),
+            '-output', 'vec.txt', '-size', '200', '-sample', '1e-4',
+            '-binary', '0', '-iter', '3', '-min_count', '1',
+        ]
+        output = check_output(args=cmd, stderr=subprocess.PIPE)
+        self.assertEqual(output, '0')
+
+
+# endclass TestWord2VecScripts
```

Comment:
Let's get rid of this too. Such closing-class comments are not really a part of our code style in Gensim.

Reply:
Removed. Only kept it because it was present before and the test above also has such a comment.